e2e/pages/compare/compare.spec.js

```diff
@@ -0,0 +1,27 @@
+import ComparePage from './compare.po';
+import Hooks from '../../utilities/hooks';
+
+let hooks, page;
+
+beforeEach(async () => {
+    page = new ComparePage();
+    hooks = new Hooks();
+    await hooks.open('#/compare/9261&9956');
+});
+
+describe('on compare page after clicking show all-', () => {
+
+    it('All 78 certification criteria should be displayed', () => {
```

Comment:
```suggestion
    it('all 78 certification criteria should be displayed', () => {
```

All of our other "it" blocks start with lower-case letters. Also, almost all of them are of the form `it('should do a thing'`, which can be read out loud as "it should do a thing", whereas this one would be read as "it all 78 criteria should be displayed" which doesn't sound right. The second point is more of a style choice, though would be more consistent with the majority of our other tests

Reply:
Fixed